docs/collecting-data/collecting-from-own-applications/c-tracker/setup/index.md

```diff
@@ -21,26 +21,49 @@ Supported operating systems:
 
 ## Installing in your project
 
-There are two ways to install the tracker in your app:
+There are three ways to install the tracker in your app:
 
-1. By including the project using cmake.
-2. By copying source files inside the `include` folder into your codebase.
+1. By adding the project into your `CMakeLists.txt` as a subdirectory.
```

Comment:
These don't seem to be in the same order as the actual ways below, it would be nice if they matched up

Reply:
hmm, they should be: 1. subdirectory, 2. import target, 3. copy files.